src/porting.h

```diff
@@ -360,6 +360,9 @@ inline u32 getDeltaMs(u32 old_time_ms, u32 new_time_ms)
 	}
 #elif defined(_WIN32)
 	inline void setThreadName(const char* name) {}
+#elif defined(__GNU__)
+//#warning "GNU/Hurd platform, pthread_setname_np not yet supported"
+inline void setThreadName(const char* name) {}
```

Comment:
Hurd doesn't support thread names though, same as Windows, so it's functionality is the same and it should be combined to reduce duplication.  Hurd obviously isn't Windows, and I wasn't implying that.  If Hurd or Windows get thread name support they can easily be given their own sections.